ansible_base/lib/utils/string.py

```diff
@@ -6,3 +6,10 @@ def make_json_safe(value):
         return value
 
     return smart_str(value)
+
+
+def is_empty(value, check_stripped=True):
```

Comment:
I'd really rather remove `check_stripped=True` from this since it's never not used.

And I'd like for pure-python utility methods like this to have some type hints and doc strings.